src/isar/config/settings.py

```diff
@@ -194,22 +194,6 @@ def __init__(self) -> None:
     # Info about robot documentation
     DOCUMENTATION: List[DocumentInfo] = Field(default=[])
 
-    # Endpoints to reach video streams for the robot
-    VIDEO_STREAMS: List[VideoStream] = Field(
```

Comment:
This deleted settings variable is also used in [RobotInfoPublisher](https://github.com/equinor/isar/blob/c8ff71cdf9bd409c3aceb59567c877251a15d71d/src/isar/services/service_connections/mqtt/robot_info_publisher.py#L25), and should be removed there as well

Reply:
Good catch :)